samples/multicore/hello_world/Kconfig.sysbuild

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-source "share/sysbuild/Kconfig"
+source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
```

Comment:
Are you sure this is needed everywhere? `source` is already relative to `srctree=${ZEPHYR_BASE}`.

Reply:
I don't see a problem with specifically showing the path, it makes it easier for someone that doesn't know it's included to open the file and see exactly what is happening